config.yaml

```diff
@@ -63,8 +69,8 @@ i18n:
     index: 
       next: "Next 〉"
       prev: "〈 Previous"
-      completion: "Thanks. Your response has been submitted."
-      completionSecondary: ""
+      completion: "Thanks!"
+      completionSecondary: "Your response has been submitted."
```

Comment:
Can we rename this `completionSubtitle` for consistency? We'll have to make it a breaking change